src/index.html

```diff
@@ -4,9 +4,310 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
```

Comment:
Need to fix with code style and check it everywhere
```suggestion
    <link
      rel="preconnect"
      href="https://fonts.gstatic.com"
      crossorigin="anonymous"
    >
```
![image](https://github.com/mate-academy/layout_catalog/assets/64750958/eb676a9a-0928-477e-8d21-6941e235e449)